getgauge/processor.py

```diff
@@ -169,11 +169,15 @@ def _get_stub_impl_content(request, response, socket):
     response.messageType = Message.FileDiff
     file_name = request.stubImplementationCodeRequest.implementationFilePath
     codes = request.stubImplementationCodeRequest.codes
-    content = read_file_contents(file_name).replace('\r\n', '\n')
-    if len(content) > 0:
+    file_content = read_file_contents(file_name)
+    if file_content != None:
+        content = file_content.replace('\r\n', '\n')
         new_line_char = '\n' if len(content.strip().split('\n')) == len(content.split('\n')) else ''
-        codes.insert(0, new_line_char)
         lastLine = len(content.split('\n'))
+        if len(content) == 0:
+            codes.insert(0, "from getgauge.python import step\n")
+        else:
+            codes.insert(0, new_line_char)
```

Comment:
It can be replaced with one-liner `if` check?
something like this -
```
prefix = "from getgauge.python import step\n" if len(content) == 0 else new_line_char
codes.insert(0, prefix)
```